README.md

```diff
@@ -13,6 +13,13 @@ enables remote device management using
 ## Supported Operating System
 
 At the moment only Linux-based systems are supported.
+On Linux system if `edgehog-device-runtime` is a `systemd` service, it can notify `systemd` its status changes.
```

Comment:
A couple of fixups:
- On Linux system -> On Linux (or On Linux systems)
- it can notify `systemd` _of_ its status changes